documents/src/pages/elements/slider.md

```diff
@@ -325,8 +324,20 @@ ef-slider[small] > ef-slider-marker[important-low] {
 ef-slider[very-small] > ef-slider-marker[important-med] {
   display: none;
 }
+input[type="radio"]:focus {
```

Comment:
The radio button has its own focus style. I don't think we need to add a focus style here.

Reply:
Thank you P'Nut. fixed.